backend/dynamo-db/template.yml

```diff
@@ -59,8 +59,10 @@ Resources:
       KeySchema:
         - AttributeName: id
           KeyType: HASH
-
       BillingMode: PAY_PER_REQUEST
+      TimeToLiveSpecification:
+        AttributeName: expires
+        Enabled: true
```

Comment:
Need to confirm the expiration period is written to the `expires` column

Reply:
As first step we are setting cookie with maxAge of one hour
```        
const DynamoDBStore = connect_dynamodb({session: session});
return session({
    store: new DynamoDBStore(options),
    secret: "keyboard cat",
    cookie: {maxAge: 1000 * 60 * 60}
});
```
And then in connect-dynamodb.js it is used to set column value
```
DynamoDBStore.prototype.getExpiresValue = function (sess) {
    var expires = typeof sess.cookie.maxAge === 'number' ? (+new Date()) + sess.cookie.maxAge : (+new Date()) + oneDayInMilliseconds;
    return Math.floor(expires / 1000);
}
```